openupgrade_scripts/scripts/l10n_es/17.0.5.4/pre-migration.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2025 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+def _pre_create_and_fill_l10n_es_is_simplified(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE account_move ADD COLUMN l10n_es_is_simplified boolean DEFAULT false
```

Comment:
You have to remove the `DEFAULT` with `DROP DEFAULT` later, or we can have unexpected behaviors.